tests/test_jobs.py

```diff
@@ -2387,6 +2505,14 @@ def test_scheduler_process_task_instances(self):
         dr = scheduler.create_dag_run(dag)
         self.assertIsNotNone(dr)
 
+        tis = dr.get_task_instances(session=session)
```

Comment:
I know that this is just a test, but can we change this using the `with` statements? For example:
```suggestion
        from airflow.utils.db import create_session
                with create_session() as session:
                    session.query(DagRun).delete()
                    session.query(TaskInstance).delete()
```
In this case we won't give a bad example.

Reply:
Sure, fixed.